I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 305, in script
    J 'อร่อยที่สุดอยู่แล้วค่ะ'
Exception: Sayer 'J' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 305, in script
    J 'อร่อยที่สุดอยู่แล้วค่ะ'
  File "C:\Users\Dell\Desktop\Project PSCP 2022\PROJECT-65\renpy-8.0.3-sdk\renpy\ast.py", line 881, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Dell\Desktop\Project PSCP 2022\PROJECT-65\renpy-8.0.3-sdk\renpy\ast.py", line 774, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'J' is not defined.

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.3.22090809
Little girl 1.0
Mon Nov 28 21:18:21 2022
